packages/components/src/components/ImportButton/ImportButton.tsx

```diff
@@ -55,7 +66,16 @@ export const ImportButton = ({ label = 'Import', onChange}: IImportButtonProps)
       }
 
       const fileArray = Array.from(files);
-      const fileContents = await Promise.all(fileArray.map((file: File) => loadJsonFile(file)));
+      const fileContents = await Promise.all(
+        fileArray.map(async (file: File) => {
+          const content = await loadJsonFile(file);
+          if (type === ImportDataType.VerifiableCredential) {
+            const processedData = await processVerifiableCredentialData(content, vcOptions?.credentialPath);
+            return { [file.name]: processedData };
```

Comment:
If we return `[file.name]` field, I tried to integrate it with our event like `processDPP`, and the error as the image below. 
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/79d17c50-e476-4be1-93bd-6fa0cd8da99c">

Reply:
The import and scanning functionality is currently only used for the processTransformationEvent. I forgot to remove the configuration in app-config, which caused some confusion here.

Reply:
@namhoang1604, but it can be used in other scenarios, right? For example, importing DCCs, DPPs, etc, into the system.